.github/workflows/node-ci.yml

```diff
@@ -28,40 +28,50 @@ jobs:
       uses: jaredly/get-changed-files@v1.0.1
       id: changed
       with:
-        format: space-delimited
-        absolute: true
+        format: json
+
+    - id: js-files
+      name: Find .js changed files
+      uses: ./.github/actions/filter-files
+      with:
+        changed-files: ${{ steps.changed.outputs.added_modified }}
+        extensions: '.js'
+
+    - id: eslint-reset
+      uses: ./.github/actions/filter-files
+      name: Files that would trigger a full eslint run
+      with:
+        changed-files: ${{ steps.changed.outputs.added_modified }}
+        files: '.eslintrc.js,yarn.lock,.eslintignore'
```

Comment:
And `package.json` or I guess that will never change without `yarn.lock` also changing?

Reply:
yeah I think yarn.lock would be enough